extensions/wikia/TemplateClassification/TemplateClassification.hooks.php

```diff
@@ -27,9 +78,13 @@ public static function register() {
 	 * @return true
 	 */
 	public function onBeforePageDisplay( \OutputPage $out, \Skin $skin ) {
-		if ( ( new Permissions() )->shouldDisplayEntryPoint( $skin->getUser(), $out->getTitle() ) ) {
-			\Wikia::addAssetsToOutput( 'tempate_classification_js' );
-			\Wikia::addAssetsToOutput( 'tempate_classification_scss' );
+		$permissions = new Permissions();
+		if ( $permissions->shouldDisplayEntryPointOnView( $skin->getUser(), $out->getTitle() ) ) {
+			\Wikia::addAssetsToOutput( 'template_classification_in_view_js' );
+			\Wikia::addAssetsToOutput( 'template_classification_scss' );
+		} elseif ( $permissions->shouldDisplayEntryPointInEdit( $skin->getUser(), $out->getTitle() ) ) {
+			\Wikia::addAssetsToOutput( 'template_classification_in_edit_js' );
+			\Wikia::addAssetsToOutput( 'template_classification_scss' );
```

Comment:
What do you think about using here just one method ```shouldDisplayEntryPoint``` and then in if statement check if it's 'edit' page.